### PR TITLE
Move docs, meta, vaxine repos in

### PR DESCRIPTION
**Pull request by define-null**
As discussed move meta, docs, vaxine into this repo without history

### CODE REVIEW

**Comment by thruflo** [comment]
Maybe extract the design folder from vaxine to the top level, plus is it worth deduping the Authors and License files?

**Comment by define-null** [comment]
@thruflo sure, could you be a bit more specific how you want me to handle license/author files? 
Move them to meta for now perhaps?

**Comment by balegas** [comment]
> Maybe extract the design folder from vaxine to the top level, plus is it worth deduping the Authors and License files?

I prefer design under docs instead of root.

**Comment by balegas** [comment]
> @thruflo sure, could you be a bit more specific how you want me to handle license/author files? Move them to meta for now perhaps?

I think all under meta can go to root (except for readme, which can disappear), and AFAIU license files are all copies and one at the root will do.

**Comment by define-null** [comment]
@thruflo @balegas:
```
$ md5sum LICENSE
2ee41112a44fe7014dce33e26468ba93  LICENSE
$ md5sum apps/vaxine/LICENSE
86d3f3a95c324c9479bd8986968f4327  apps/vaxine/LICENSE
$ md5sum docs/LICENSE
86d3f3a95c324c9479bd8986968f4327  docs/LICENSE
```

meta/CLA.md is different as well.

**Review by v0idpwn — COMMENTED** [review]
Structure looks good to me. Do you plan to have elixir building/testing on this PR?

**Comment by define-null** [comment]
As long as we keep `elixir` purely for the client libs, and `erlang` for the server - we may have different targets to build vaxine release and vax lib .But if we decide we would like to have elixir code coexist with erlang on the server side - we most likely would have to migrate to `mix` as support for elixir in the umbrella apps looks limited from rebar3 perspective. For now we can just have multiple build targets as client and server code does not depend on each other.

So to answer your question probably not in this PR

File: meta/README.md [hunk 1]
```diff
@@ -1,10 +0,0 @@
-
-# Vaxine Community Guidelines
```
Comment (thruflo) [inline meta/README.md]: Be nice to copy this into the main README.

**Review by thruflo — APPROVED** [review]

**Review by v0idpwn — APPROVED** [review]

**Pull request merged**